src/services/giant_api.rs

```diff
@@ -116,13 +124,19 @@ pub fn get_or_insert_ingestion(
 }
 
 // Returns a maximum of 500 blobs per request
-pub fn get_blobs_in_collection(giant_uri: &str, collection: &str) -> Result<Vec<Blob>, CliError> {
+pub fn get_blobs_in_collection(
+    giant_uri: &str,
+    collection: &str,
+    filter: &ListBlobsFilter,
+) -> Result<Vec<Blob>, CliError> {
     let client = get_client(giant_uri)?;
     let encoded_collection = encode(collection);
-    let url = format!("{giant_uri}/api/blobs?collection={encoded_collection}");
-
+    let in_multiple = match filter {
+        ListBlobsFilter::All => "",
+        ListBlobsFilter::InMultiple => "&inMultiple=true",
+    };
+    let url = format!("{giant_uri}/api/blobs?collection={encoded_collection}{in_multiple}");
```

Comment:
Minor point, but if you prefer you can use the `Url` stuct which is reexported from `reqwest` to do this with less manual string manipulation.

It might look something like this (I wrote this off the top of my head so it might be wrong in some way 😬 )
```rust
let mut url = Url::from(format!("{giant_uri}/api/blobs"));
let in_multiple = filter == ListBlobsFilter::InMultiple;
url.query_pairs_mut()
    .append_pair("collection", collection)
    .append_pair("inMultiple", in_multiple);
```

Reply:
I've got another PR coming for a `CountBlobs` subcommand. Will try this out in that one